.changeset/spicy-numbers-deny.md

```diff
@@ -0,0 +1,20 @@
+---
+"@commercetools/composable-commerce-test-data": patch
+---
+
+Added new submodel to the ProductSelection model:
```

Comment:
```suggestion
Added new submodel to the ProductOfSelection model:
```

Reply:
But the model we are adding submodel to is called `ProductSelection`, `ProductOfSelection` is the name of the submodel.

Reply:
You're totally right. Sorry.